src/Features/header/components/MobileApplicationPromo/MobileApplicationPromo.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { render, screen } from '@testing-library/react';
+
+import { MobileApplicationPromo } from './MobileApplicationPromo';
+
+describe('Test MobileApplicationPromo component', () => {
+  test('Should render correctly', () => {
+    render(<MobileApplicationPromo />);
+
+    expect(screen.getByText(/Установите специальное/i)).toBeInTheDocument();
+    expect(screen.getByText(/Мобильное приложение/i)).toBeInTheDocument();
+    expect(screen.getByText(/Для удобной онлайн-записи/i)).toBeInTheDocument();
+    expect(screen.getByText(/Google Play/i)).toBeInTheDocument();
+    expect(screen.getByText(/Google Play/i)).toHaveClass('text-sm');
+    expect(screen.getByText(/Google Play/i)).toHaveClass('font-bold');
+    expect(screen.getByText(/App Store/i)).toBeInTheDocument();
+    expect(screen.getByText(/App Store/i)).toHaveClass('text-xs');
```

Comment:
You can check classes but this is optional Most of the time if we want to check classes and styles in our test, we should use snapshot testing. For now, we will leave it out of the scope